docker/Dockerfile.dev

```diff
@@ -0,0 +1,12 @@
+FROM python:3.11-bookworm
+
+RUN apt-get update -y && \
```

Comment:
Why not just use the upstream Python image?

Reply:
I'm not 100% sure what upstream is referring to. If it's referring to the Docker Image, I think I'm already using it. I just used a Debian-based tag.
Did you mean just use `debian:bookworm` and just install Python via `apt-get`?